src/SWP/Bundle/UserBundle/Resources/config/routing.yml

```diff
@@ -2,14 +2,26 @@ swp_user:
     resource: "@SWPUserBundle/Controller"
     type:     annotation
 
-fos_user_registration_check_email:
-    path: /register/check-email
-    defaults: { _controller: fos_user.registration.controller:checkEmailAction }
+#swp_user_registration_check_email:
+#    path: /register/check-email
```

Comment:
why is this redirect required, I mean it would only be necessary if someone was doing password resetting process during the update to new userbundle version (same request) so it's quite little time window when it could be necessary.
Or do are you thinking about old template compatibility.

Reply:
old template compatibility it should be backward compatible

Reply:
this comment is already removed